.github/workflows/push-on-develop.yml

```
@@ -82,10 +96,76 @@ jobs:
+        with:
+          images: ${{ env.REGISTRY_IMAGE }}
+          tags: ${{ env.REGISTRY_IMAGE }}:${{ matrix.tag }}
+
+      - name: Log in to Docker Hub 🔐
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+
+      - name: Create manifest list and push 📦
+        run: |
```

actionlint

shellcheck reported issue in this script: SC1009:info:6:30: The mentioned syntax error was in this simple command

(shellcheck)